Files.Launcher/MessageHandlers/MessageHandler.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO.Pipes;
+using System.Threading.Tasks;
+
+namespace FilesFullTrust.MessageHandlers
+{
+    public interface MessageHandler : IDisposable
```

Comment:
Since `MessageHandler` is an interface, it should start with `I`. `IMessageHandler`